src/app.js

```diff
@@ -344,7 +345,7 @@ function run () {
           fileManager.switchFile(filename)
         }
         if (lineColumn.start && lineColumn.start.line && lineColumn.start.column) {
-          editor.gotoLine(lineColumn.start.line, lineColumn.start.column + 1)
+          editor.gotoLine(lineColumn.start.line, lineColumn.end.column + 1)
```

Comment:
is that fixing a bug?

Reply:
Nope, actually previous jumpTo throws you at the beginning of the line and deselects the chosen keyword => this then makes next/previous not work anymore. 

So, I now made jumpTo jump to the right line, but directly on the word, so keyword stays selected.

Do you think any other solution would be better?

Reply:
I don't have any yet..